webpack.js

```diff
@@ -12,51 +15,30 @@ if (isTesting) {
 	console.debug('TESTING MODE ENABLED')
 }
 
-const config = {
-	module: {
-		rules: [
-			{
-				// vue-plyr uses .mjs file
-				test: /\.m?js$/,
-				loader: 'babel-loader',
-				exclude: BabelLoaderExcludeNodeModulesExcept([
-					'@nextcloud/dialogs',
-					'@nextcloud/event-bus',
-					'camelcase',
-					'fast-xml-parser',
-					'hot-patcher',
-					'semver',
-					'vue-plyr',
-					'webdav',
-					'toastify-js',
-				]),
-				options: {
-					plugins: ['add-module-exports'],
-					presets: [
-						/**
-						 * From "add-module-exports" documentation:
-						 * "webpack doesn't perform commonjs transformation for
-						 * codesplitting. Need to set commonjs conversion."
-						 */
-						['@babel/env', { modules: 'commonjs' }],
-					],
-				},
-			},
-		],
-	},
-	plugins: [
-		new webpack.DefinePlugin({
-			isTesting,
-			PLYR_ICONS: JSON.stringify(plyrIcons),
-		}),
-	],
-}
+// vue-plyr uses .mjs file
+webpackRules.RULE_JS.test = /\.m?js$/
+webpackRules.RULE_JS.exclude = BabelLoaderExcludeNodeModulesExcept([
+	'@nextcloud/dialogs',
+	'@nextcloud/event-bus',
+	'camelcase',
+	'fast-xml-parser',
+	'hot-patcher',
+	'semver',
+	'vue-plyr',
+	'webdav',
+	'toastify-js',
+])
 
-const mergedConfigs = merge(config, webpackConfig)
+// Replaces rules array
+webpackConfig.module.rules = Object.values(webpackRules)
 
-// Remove default js rule
-const jsRuleIndex = mergedConfigs.module.rules.findIndex(rule => rule.test.toString() === '/\\.js$/')
-mergedConfigs.module.rules.splice(jsRuleIndex, 1)
+// Add custom plugins
+webpackConfig.plugins.push(...[
+	new webpack.DefinePlugin({
+		isTesting,
+		PLYR_ICONS: JSON.stringify(plyrIcons),
+	}),
+	new NodePolyfillPlugin(),
```

Comment:
Not needed anymore once https://github.com/nextcloud/webpack-vue-config/pull/88 is in and released